aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/CqChildEditConfigWriter.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.core.util;
+
+import com.exadel.aem.toolkit.api.annotations.editconfig.ChildEditConfig;
+import com.exadel.aem.toolkit.api.annotations.widgets.common.XmlScope;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.transform.Transformer;
+import java.util.Arrays;
+
+/**
+ * The {@link PackageEntryWriter} implementation for storing author UI aspects for child components that do not define
```

Comment:
... for defining behaviour of this component's chilren in Touch UI environment. This serves as a bulk setting equivalent of {@code cq:EditConfig} node for the immediate children of the current component unless they have their own {@code cq:EditConfig} nodes

@liubou-masiuk pls check